backend/corpora/dataset_processing/process.py

```diff
@@ -389,10 +392,21 @@ def process_cxg(local_filename, dataset_id, cellxgene_bucket):
     bucket_prefix = get_bucket_prefix(dataset_id)
     cxg_dir, status = convert_file_ignore_exceptions(make_cxg, local_filename, "Issue creating cxg.")
     if cxg_dir:
-        copy_cxg_files_to_cxg_bucket(cxg_dir, bucket_prefix, cellxgene_bucket)
+        s3_uri = copy_cxg_files_to_cxg_bucket(cxg_dir, bucket_prefix, cellxgene_bucket)
         metadata = {
             "explorer_url": join(DEPLOYMENT_STAGE_TO_URL[os.environ["DEPLOYMENT_STAGE"]], dataset_id + ".cxg", "")
         }
+        with db_session_manager() as session:
+            logger.info(f"Updating database with  {DatasetArtifactFileType.CXG}.")
```

Comment:
Could add the `s3_uri` here which could be helpful for debugging exactly which one to look into.